### ISSUE TITLE
Google Analytics

### ISSUE THREAD

**Issue by TangoYankee** [openoakland/woeip#27]
## Description
We want to be able to track visitor behavior across the site via tracking analytics. Some options:

- Google Analytics
- [Open Web Analytics](http://www.openwebanalytics.com/) (free)
- Fathom ([paid](https://usefathom.com/) and [free Lite version](https://github.com/usefathom/fathom))
- Plausible ([paid](https://github.com/plausible/analytics), w/ an alpha-phase free self-hosted version)

## Considerations
- Account should be tied to woaq@openoakland.com
- Dashboard needs (e.g. what goal paths need to be set up)

**Comment by theecrit** [comment]
Let’s also investigate Google Tag Manager when we tackle this; it may provide more flexibility as the project evolves.

**Comment by theecrit** [comment]
@TangoYankee Do you think we need this for MVP? If MVP is really an internal staff testing period, I wonder if maybe we can punt this to post-MVP, when it gets rolled out to actual volunteers for real use.

**Comment by TangoYankee** [comment]
> Do you think we need this for MVP? 

I don't think it should be labeled MVP. 

At the same time, it's a relatively light lift. It could be accomplished independently and in parallel with MVP issues.

**Comment by theecrit** [comment]
Yeah, I feel like there are a handful of issues that don't *need* to be in MVP but could also be good issues for new folks to knock out as a way of getting familiar with the project. This may not be one of those (given that it requires admin-type access for the GA account), but throwin' it out there.

**Comment by theecrit** [comment]
Added some issue description/detail

**Comment by theecrit** [comment]
Assigning to MVP, as MVP will be a live site.
